.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
   django-unit-tests:
     strategy:
       matrix:
-        django: [3.2, 3.1.8, 3.0.14, 2.2.20]
+        django: ["4.0", "3.2", "2.2"]
```

Comment:
shall we remove official support for 2.2 since it is not LTS anymore?

Reply:
what about 3.0 and 3.1?

Reply:
2.2 is LTS
3.0 was dropped a few month ago.
3.1 reached end of life today.

https://www.djangoproject.com/download/